rds/index.js

```diff
@@ -318,6 +325,8 @@ class StatementBuilder {
         return `${startGrouping}${this.quoteChar}${columnName}${this.quoteChar} LIKE ${value}${endGrouping}`;
       case "notContains":
         return `${startGrouping}${this.quoteChar}${columnName}${this.quoteChar} NOT LIKE ${value}${endGrouping}`;
+      case "attributeExists":
+        return `${startGrouping}${this.quoteChar}${columnName}${this.quoteChar} IS ${value? "NOT " : ""}NULL${endGrouping}`;
```

Comment:
I'm not a super fan of this ternary operator, but it's quite common in JS so I'm ok with it.